src/ports/items/utils.ts

```diff
@@ -183,8 +225,54 @@ export function getItemsQuery(filters: ItemFilters, isCount = false) {
     where.push('soldAt_not: null')
   }
 
-  if (isWearableSmart) {
-    where.push(`itemType: smart_wearable_v1`)
+  if (!category || category === NFTCategory.WEARABLE) {
+    if (wearableCategory) {
+      where.push(`searchWearableCategory: ${wearableCategory}`)
+    }
+
+    if (isWearableHead) {
+      where.push('searchIsWearableHead: true')
+    }
+
+    if (isWearableAccessory) {
+      where.push('searchIsWearableAccessory: true')
+    }
+
+    if (wearableGenders && wearableGenders.length > 0) {
+      const hasMale = wearableGenders.includes(WearableGender.MALE)
+      const hasFemale = wearableGenders.includes(WearableGender.FEMALE)
+
+      if (hasMale && !hasFemale) {
+        where.push(`searchWearableBodyShapes: [BaseMale]`)
+      } else if (hasFemale && !hasMale) {
+        where.push(`searchWearableBodyShapes: [BaseFemale]`)
+      } else if (hasMale && hasFemale) {
+        where.push(`searchWearableBodyShapes_contains: [BaseMale, BaseFemale]`)
+      }
+    }
+
+    if (isWearableSmart) {
+      where.push(`itemType: smart_wearable_v1`)
+    }
+  }
+
+  if (!category || category === NFTCategory.EMOTE) {
```

Comment:
It is intended that if no categoy is provided, both wearables and emotes will be included right?